sonic-psud/scripts/psud

```diff
@@ -79,11 +80,25 @@ def _wrapper_get_psus_status(psu_index):
 
 def psu_db_update(psu_tbl, psu_num):
     for psu_index in range(1, psu_num + 1):
+        psu_presence_state = psuutil.get_psu_presence(psu_index)
```

Comment:
wrapper functions _wrapper_get_psus_presence() and _wrapper_get_psus_status() are intend to make psud can work both with platform plugin and new platform API.  Here your change will break psud on the platform which only supports platform API.

Reply:
Agree with @keboliu. The wrapper functions need to be used until all platforms have migrated over to the new platform API.